lib/src/indexer.dart

```diff
@@ -19,8 +20,8 @@ Future<Index> indexPackage(
       projectRoot: 'file:/' + dirPath,
       textDocumentEncoding: TextEncoding.UTF8,
       toolInfo: ToolInfo(
-        name: 'scip-dart',
-        version: '0.0.1',
+        name: 'scip_dart',
```

Comment:
while the repo is called `scip-dart`, due to pub rules, the package is called `scip_dart`

Officially the name should be `scip_dart`, so reflect this in the tool info